tools/SetupFlow/DevHome.SetupFlow/Models/InstallPackageTask.cs

```diff
@@ -282,11 +282,11 @@ private void ReportAppSelectedForInstallEvent()
 
     private void ReportAppInstallSucceededEvent()
     {
-        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_InstallSucceeded", LogLevel.Critical, new AppInstallEvent(_package.Id, _package.CatalogId), _activityId);
+        TelemetryFactory.Get<ITelemetry>().Log("AppInstall_InstallSucceeded", LogLevel.Critical, new AppInstallResultEvent(_package.Id, _package.CatalogId), _activityId);

```

Comment:
For telemetry prefer to use one event name and to use a variable to separate the actions/result/operations.  For example the event name could be `AppPage_AppInstall_InstallEvent` and `AppInstallResultEvent` can have a `public bool DidSucceed` variable.

Reply:
Same comment as above